build.sbt

```diff
@@ -155,8 +157,8 @@ lazy val webkit =
         commons_fileupload,
         rhino,
         servlet_api,
-        specs2.copy(configurations = Some("provided")),
-        specs2Matchers.copy(configurations = Some("provided")),
+        specs2Main,
+        specs2MatchersMain,
```

Comment:
The new one puts it on the classpath, while the old one marked it provided (~= not on the classpath via sbt, but on the classpath via external means?). Not fully following the difference in what we were doing vs what we're doing now.

Reply:
Ahhhhh right you are.